common/changes/pivotHeadersOnly_2017-04-07-10-40.json

```diff
@@ -0,0 +1,20 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Pivot: new props headersOnly and getTabId.",
```

Comment:
Maybe something along the lines of "Pivot: Allow rendering PivotItem headers without content"?